proposals/0035-linalg-matrix.md

```diff
@@ -1725,9 +1783,8 @@ void CoopVec() {
       Matrix<ComponentType::F16, 16, 16, MatrixUse::A, MatrixScope::Thread>;
 
   vector<float16_t, 16> Vec = (vector<float16_t, 16>)0;
-  MatrixATy MatA = MatrixATy::Load(
-      MBuf, 0, /* Row stride = number of columns * element size */ 16 * 4,
-      MatrixLayout::RowMajor);
+  MatrixATy MatA = MatrixATy::Load<MatrixLayout::RowMajor>(
```

Comment:
Is there a particular reason the Layout is a template parameter for the CoopVec use case but a formal parameter for the WaveMatrix use case? Having them different feels like an API wrinkle